README.md

```diff
@@ -327,8 +327,82 @@ Templates in React are written inside the JavaScript file using the [JSX languag
 
 # Filters
 
-> TODO
-> Angular's pipe character (|) 
+### AngularJS
+
+AngularJS provides filters to transform data for transform of data.<br />
+There are several built-in filters to use or you can make your own custom filters as well.<br />
+Filters can applied to view template using following syntax :  `{{ expression | filter }}`<br />
+Chaining of filters is also possible :   `{{ expression | filter1 | filter2 }}`<br />
+
+Built-in Filters are as follows : <br />
+  -`currency` Format a number to a currency format.<br />
+  -`date` Format a date to a specified format.<br />
+  -`filter` Select a subset of items from an array.<br />
+  -`json` Format an object to a JSON string.<br />
+  -`limitTo` Limits an array/string, into a specified number of elements/characters.<br />
+  -`lowercase` Format a string to lower case.<br />
+  -`number` Format a number to a string.<br />
+  -`orderBy` Orders an array by an expression.<br />
+  -`uppercase` Format a string to upper case.<br />
+
+
+
+Code Snippet of LowerCase Filter : 
+
+```
+  class MyFilter {
+   
+  static filter( value ){
+    return value.toLowerCase();
+  }
+   
+}
+ 
+export default MyFilter.filter;
+```
+
+Basic Implementation in html :
+
+```
+<div ng-app="myApp" ng-controller="caseCtrl">
+
+<h1>{{txt | lowercase}}</h1>
+
+</div>
+
+<script>
+var app = angular.module('myApp', []);
+app.controller('caseCtrl', function($scope) {
+    $scope.txt = "Hello World!";
+});
+</script>
+```
+
+
+Code Snippet of Custom Filter : 
```

Comment:
Please leave just one empty space between text / heading / code snippet. Markdown will ignore it either way.